indico/modules/events/sessions/forms.py

```diff
@@ -40,3 +41,21 @@ class SessionForm(IndicoForm):
                                       description=_('Specify text and background colours for the session.'))
     is_poster = BooleanField(_('Poster session'), widget=SwitchWidget(),
                              description=_('Whether the session is a poster session.'))
+
+
+class EmailSessionPersonsForm(IndicoForm):
+    from_address = SelectField(_('From'), [DataRequired()], choices=[(1, 1)])
+    subject = StringField(_('Subject'), [DataRequired()])
+    email_recipients = IndicoStaticTextField(_('List of the recipients'), text='')
```

Comment:
I'd put it below the body and just call it "Recipients". And since text is required atm, maybe add a `''` default to the `pop()` in the field class? doesn't seem to make sense to require it...